### PR TITLE
Move test file to keep consistency

### PR DESCRIPTION
**Pull request by ujovlado**
Proposed changes:

- Join multiple test files to one
- Move test file next to module
- Fix assertion order

### CODE REVIEW

File: src/scripts/modules/ex-db-generic/storeProvisioning.test.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+describe('shouldDestinationHaveOldFormat', function() {
+
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  it('it should have new format (no tables)', function() {
```
Comment (matejrokos) [inline src/scripts/modules/ex-db-generic/storeProvisioning.test.js]: Please provide better expectation description like `Expect new format (false) when there are no tables in parameters.`

File: src/scripts/modules/ex-db-generic/storeProvisioning.test.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          return Immutable.List();
+        }
+      };
+    });
+    const store = require('./storeProvisioning')
+      .createStore('keboola.ex-db-mysql', '333289236222');
+    expect(store.shouldDestinationHaveOldFormat('in.c-keboola-ex-db-mysql')).toEqual(false);
+  });
+
+  it('it should have old format (only old tables)', function() {
```
Comment (matejrokos) [inline src/scripts/modules/ex-db-generic/storeProvisioning.test.js]: Please provide better expectation description.

File: src/scripts/modules/ex-db-generic/storeProvisioning.test.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          ]);
+        }
+      };
+    });
+    const store = require('./storeProvisioning')
+      .createStore('keboola.ex-db-mysql', '333289236222');
+    expect(store.shouldDestinationHaveOldFormat('in.c-keboola-ex-db-mysql')).toEqual(true);
+  });
+
+  it('it should have new format (custom destination)', function() {
```
Comment (matejrokos) [inline src/scripts/modules/ex-db-generic/storeProvisioning.test.js]: Please provide better expectation description. It should be clear what is expected and when.

**Comment by ujovlado** [comment]
@matejrokos opravene

**Pull request merged**